ports/servoshell/egl/host_trait.rs

```diff
@@ -74,8 +74,6 @@ pub trait HostTrait {
     fn on_media_session_playback_state_change(&self, state: MediaSessionPlaybackState);
     /// Called when the media session position state is set.
     fn on_media_session_set_position_state(&self, duration: f64, position: f64, playback_rate: f64);
-    /// Called when devtools server is started
-    fn on_devtools_started(&self, port: Result<u16, ()>, token: String);
```

Comment:
What is the intended way to disable prompts for devtools now? 
For ohos it is a problem, since currently we need to patch out the prompt check (since interactive prompts are not implemented on ohos (and android too I think).

Reply:
The default behavior is [to accept](https://github.com/servo/servo/blob/dd49bf38bf35c401991b553526e914efe1706ee9/ports/servoshell/egl/app_state.rs#L106) the devtools connection requests.